rwengine/src/script/modules/ObjectModule.cpp

```diff
@@ -1129,6 +1072,20 @@ bool game_character_in_range(const ScriptArguments& args)
 	return true;
 }
 
+void game_create_weapon_pickup(const ScriptArguments& args)
+{
+	glm::vec3 pos (args[3].real, args[4].real, args[5].real);
+	int id = args.getModel(0);
+	int type = args[1].integer;
+	int ammo = args[2].integer;
+	std::cout << "weapon: " << glm::to_string(pos) << " " << id << std::endl;
+	RW_UNUSED(ammo);
+	RW_UNIMPLEMENTED("game_create_weapon_pickup(): ammo count");
+
+	PickupObject* pickup = args.getWorld()->createPickup(pos, id, type);
+	*args[6].globalInteger = pickup->getGameObjectID();
+}
+
```

Comment:
Maybe this should be a seperate commit? (one for improving the old opcode, one for adding the new one)